cvat/apps/organizations/serializers.py

```diff
@@ -58,11 +61,12 @@ class InvitationReadSerializer(serializers.ModelSerializer):
     organization = serializers.PrimaryKeyRelatedField(
         queryset=Organization.objects.all(),
         source='membership.organization')
+    organization_info = BasicOrganizationSerializer(source='membership.organization')
```

Comment:
@klakhov , let's meet and discuss some changes. I'm not sure why we need duplication of data here.

Reply:
Can we solve the problem with organization and organization_info in the PR?

Reply:
OK. Let's keep as is at the momemt